cloudigrade/account/models.py

```diff
@@ -85,3 +88,44 @@ def product_identifier(self):
 
         """
         return f'RHELX-{self.ec2_ami_id}-{self.instance_type}'
+
+
+def create_initial_instance_events(account, instances_data):
```

Comment:
I don't think this belongs here, as this has nothing to do with defining a model.

Reply:
I agree, and I was pondering a name for a module where this would belong when I decided to just stick it here for the time being. Any new module name suggestions, @ilyavw ?

Reply:
`account.util`?

Reply:
👌